app/src/components/directives/my-nav-link/myNavLink.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../style/themes/style-assets';
```

Comment:
For sass import includes, just use name like `style-assets` and then set include paths in webconfig. Remember this is a `scss` file, not a `js` file so `@import` is totally different than `import`.